nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/TestListenTCP.java

```diff
@@ -156,7 +156,7 @@ public void testTLSClientAuthRequiredAndClientCertNotProvided() throws Initializ
         messages.add("This is message 5\n");
 
         // Make an SSLContext that only has the trust store, this should not work since the processor has client auth REQUIRED
-        Assert.assertThrows(SSLException.class, () ->
```

Comment:
This is a subtle change that works differently based on the specific update revision of Java 8.  More recent versions, after Update 271 in general, should throw an `SSLException` whereas older versions will throw an IOException.  Did you get a test failure?  The recent GitHub CI checks failed with a Checkstyle violation due to the unused import of `SSLException`.  Since `SSLException` is a subclass of `IOException`, your change to the more general exception seems like the better way to go, just remove the unused import and that should work.